styles/SpringEffect.ts

```diff
@@ -0,0 +1,30 @@
+export const SpringEffect = {
+  initial: {
+    opacity: 0,
+    scale: 0.5,
+    transition: {
+      duration: 0.3,
+      ease: [0, 0.71, 0.2, 1.01],
+      scale: {
+        type: "spring",
+        damping: 7,
+        stiffness: 80,
+        restDelta: 0.001,
+      },
+    },
+  },
+  animate: {
+    opacity: 1,
+    scale: 1,
+    transition: {
+      duration: 0.3,
+      ease: [0, 0.71, 0.2, 1.01],
```

Comment:
```suggestion
export const defaultEasing = [0, 0.71, 0.2, 1.01];
export const SpringEffect = {
  initial: {
    opacity: 0,
    scale: 0.5,
    transition: {
      duration: 0.3,
      ease: defaultEasing,
      scale: {
        type: "spring",
        damping: 7,
        stiffness: 80,
        restDelta: 0.001,
      },
    },
  },
  animate: {
    opacity: 1,
    scale: 1,
    transition: {
      duration: 0.3,
      ease: defaultEasing,
     }
 ```
 nit
 이런식으로 관리해도 좋을 것 같아용!

Reply:
네 좋은거 같아요 🤧